src/index.html

```diff
@@ -4,19 +4,107 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <title>Page Title</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
     />
-    <title>Moyo header</title>
+
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo-link"
+      >
+        <img
+          src="images\moyo_1.png"
+          alt="MOYO logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#link-1"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-2"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-3"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#link-4"
+              class="nav__link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-5"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-6"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-7"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#link-8"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
Remember to add empty lines between multiline sibling blocks of HTML. This will improve the readability of your code. You should add empty lines between the `<li>` elements in your navigation.